src/bin/eolian/main.c

```diff
@@ -492,47 +493,101 @@ _write_deps(const Eolian_State *eos, const char *ofname, const char *ifname,
    return ret;
 }
 
-int
-main(int argc, char **argv)
+typedef struct _Cmd_Line_Options {
+   Eina_Bool scan_system;
+   Eina_List *includes;
+   int pret;
+   int gen_what;
+   char *basen;
+   char *outs[sizeof(_dexts) / sizeof(void *)];
+   const char *input;
+} Cmd_Line_Options;
+
+static int
+_process_options(int argc, char** argv, Cmd_Line_Options* options);
```

Comment:
Two things not so important: (1) declarations should be done in one line and (2) you may prefer `type *var` over `type* var`:
```suggestion
static int _process_options(int argc, char **argv, Cmd_Line_Options *options);
```
Possibly it occur in other places of the code too.

Reply:
OH, right, this `type *var` is also another EFL standard I forgot.

Reply:
Push'd.